tower-http/src/compression/layer.rs

```diff
@@ -78,6 +88,21 @@ impl CompressionLayer {
         self.accept.set_br(false);
         self
     }
+
+    /// Configures the minimum size at which the layer starts compressing response
+    /// bodies.
+    ///
+    /// Any response smaller than `min` will not be compressed. The response size
+    /// is determined by inspecting [`Body::size_hint()`] and the `content-length`
+    /// header.
+    ///
+    /// Passing `0` makes the layer compress every response.
+    ///
+    /// The default is 32 bytes.
+    pub fn min_size(mut self, min: u16) -> Self {
```

Comment:
What do you think about making this an `Option<u16>` and using `None` to mean "compression everything"? I generally find that a bit more explicit.

Reply:
Should probably be `impl Into<Option<u16>>` which will allow passing `u16`s directly and thus a bit more ergonomic.